client/web/src/utilities/input.ts

```diff
@@ -6,7 +8,11 @@ export function shouldRedirectKeyboardEventToBackend(e: KeyboardEvent): boolean
 	if (target.nodeName === "INPUT" || target.nodeName === "TEXTAREA" || target.isContentEditable) return false;
 
 	// Don't redirect a fullscreen request
-	if (e.key.toLowerCase() === "f11") return false;
+	if (e.key.toLowerCase() === "f11" && e.type === "keydown" && !e.repeat) {
```

Comment:
Would it be a good idea to lowercase `e.type`? I'm sure it's consistent in practice, but I figure the same is also true of `e.key`, so it seems reasonable to lowercase either both or neither

Reply:
The JS spec guarantees that capitalization of its event names. I'm less certain if that's the case for keyboard input names.